addons/website_jitsi/views/chat_room_templates.xml

```diff
@@ -8,7 +8,8 @@
             t-att-data-auto-open="auto_open"
             t-att-data-check-full="check_full"
             t-att-data-attach-to="attach_to"
-            t-att-data-default-username="default_username">
+            t-att-data-default-username="default_username"
+            t-att-data-jitsi-server="env['ir.config_parameter'].sudo().get_param('website_jitsi.jitsi_server')">
```

Comment:
Hmm.

How about a "jitsi_server_domain" field that is computed on the room model?
And the compute would fallback to the config parameter?

Not a fan of calling "env" and "sudo" in templates, and I don't see a lot of occurrences of that kind of things in the code currently.

Reply:
@awa-odoo But it means that we will also need to change the way the widget is called
```xml
<t t-call="website_jitsi.chat_room_join_button">
    <t t-set="_classes" t-value="'d-none'"/>
    <t t-set="room_name" t-value="sponsor.room_name"/>
    <t t-set="chat_room_id" t-value="sponsor.chat_room_id.id"/>
    <t t-set="auto_open" t-value="1"/>
    <t t-set="attach_to" t-value="'#o_wsponsor_jitsi_iframe'"/>
    <t t-set="check_full" t-value="option_can_edit"/>
    <t t-set="jitsi_server_domain" t-value="sponsor.jitsi_server_domain"/>
</t>
```

But it has the advantage to be able to have a different server for the exhibitor and for the meeting room e.g....

Don't know what is better, what do you think ?

Reply:
Hmm I was thinking of putting the computed field on the room model itself.
To have something like: `<t t-set="chat_room_id" t-value="sponsor.chat_room_id.jitsi_server_domain"/>`.
So the calls on both sides will be very similar.

I think it's sufficient for now to have only one server switch for both use cases, right?

Reply:
(Oops, made a typo in my comment, I fixed it)

Reply:
Ah right (I've made a mistake in my example), ok, let's go for that :)

Reply:
(I update the pad with the templates to update)